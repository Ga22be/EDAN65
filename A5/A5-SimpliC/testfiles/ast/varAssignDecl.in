int a() {
    int c = 3;
}