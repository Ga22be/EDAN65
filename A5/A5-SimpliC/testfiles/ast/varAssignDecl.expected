Program
  List
    Function
      IdDecl ID="a"
        IntType
      List
      List
        DeclAssignment
          IdDecl ID="c"
            IntType
          Numeral NUMERAL="3"
